src/components/narrative/MobileNarrativeDisplay.js

```diff
@@ -153,13 +153,25 @@ class MobileNarrativeDisplay extends React.Component {
           {this.renderVizCards(this.state.contentHeight)}
           {this.renderMainMarkdown()}
         </MobileContentContainer>
-        <MobileBannerBottom height={bannerHeight} onClick={this.goToNextPage}>
+        <this.NextButton>
           Next
-        </MobileBannerBottom>
+        </this.NextButton>
       </>
     );
   }
 
+  NextButton = (props) => (
+    <MobileBannerBottom height={this.state.bannerHeight} onClick={this.goToNextPage}>
+      {props.children}
+    </MobileBannerBottom>
+  )
+
+  PreviousButton = (props) => (
```

Comment:
This is cool - I didn't know fat-arrow functions would bind `this` when used inside classes.

Reply:
Me neither!